rest-api-spec/src/main/resources/rest-api-spec/test/cat.indices/10_basic.yaml

```diff
@@ -71,6 +71,41 @@
                 $/
 
 ---
+"Test cat indices using health status":
+  - do:
+      indices.create:
+        index: foo
+        body:
+          settings:
+            number_of_shards: "1"
+            number_of_replicas: "0"
+  - do:
+      indices.create:
+        index: bar
+        body:
+          settings:
+            number_of_shards: "1"
+            number_of_replicas: "1"
```

Comment:
I think this assumes that no rest test runner runs against multiple nodes. I think the some of the client test runners do though, so this index will be green for them.